itsnameissteven/metAPI
# Feature/error handling

## What does this PR do?

- Intalls Cypress
- Adds error handling function to the APIcalls file
- Adds 400 and 500 error handling logic
- Introduces Switch components to the App component render method

## How should it be tested in the UI?

- Cypress was installed as a dev dependency. Not sure if you'll need to install it locally when you fetch this branch
- Use Cypress to stub network requests and see error message response
- You can also test bad paths by manually navigating to `localhost3000/anything` to test the Switch catch all.
